recipes/spirv-headers/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.5.1.corrected":
```

Comment:
I think the version should be 1.5.1 as they state in their repo the following:
```
This has version numbers corrected from 1.4 to 1.5.
```

Reply:
@SSE4 @uilianries opinions welcomed

Reply:
I can set that up, but I will have to make a separate python script (spirv-headers/1.5.1, instead of spirv-headers/all) for this specific package because when the tar archive is extracted, the folder that gets created is SPIRV-Headers-1.5.1.corrected, so this specific version will need an extra line of code to make sure it renames the folder properly.

Or I can add this this to the source method for the "all" script 
```
       if self.version == "1.5.1":
            extracted_dir = extracted_dir + ".corrected"
```

Reply:
yes, I was thinking about the second approach with the version conditional

Reply:
Use the second version. Indeed, I'll not be happy looking for a package like foo/1.0.0.<someextrasuffix>

Reply:
Sounds good! Updated